cdap-docs/tools/doc-change.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#  Copyright © 2016 Cask Data, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+# Changes a doc set in place, modifying the headers.
+# Follows these rules:
+#
+# - Looks for any empty title tags (<tile></title>) and adds the version in between, such as
+#   <tile>CDAP 3.6.0 Documentation</title>
+#
+# For pages that are to be on docs.cask.co (non-GitHub pages):
+# 
+# - If the canonical numbered version:
+#   no changes
+#   If basename is not the numbered version to be used for empty title tags, use
```

Comment:
So more like "if the version derived from the basename of $foo is not the numbered version"...